awx/main/tests/data/inventory/plugins/gce/files/gcp_compute.yml

```diff
@@ -0,0 +1,46 @@
+auth_kind: serviceaccount
+compose:
+  ansible_ssh_host: networkInterfaces | json_query("[0].accessConfigs[0].natIP")
+  gce_description: description if description else None
+  gce_id: id
+  gce_machine_type: machineType
+  gce_metadata: metadata.get("items", []) | items2dict(key_name="key", value_name="value")
+  gce_name: name
+  gce_network: networkInterfaces | json_query("[0].network.name")
```

Comment:
Alternatively use `networkInterfaces[0]['network']['name']` and negate the need of `json_query`